tests/Sandpit/Exscientia/FreeEnergy/test_alchemical_free_energy.py

```diff
@@ -62,7 +63,7 @@ def amber_data(tmp_path_factory):
     @pytest.fixture(scope="class")
     def gmx_complex(gmx_data):
         outdir = gmx_data
-        complex, _ = BSS.FreeEnergy.AlchemicalFreeEnergy.analyse(
+        complex, _ = BSS.FreeEnergy.Relative.analyse(
```

Comment:
Have you resolved the conflicts the correct way? It seems that `AlchemicalFreeEnergy` has been replaced by `Relative` in many places.

Also, the Sire packages are not yet available so the CI won't currently run.

Reply:
Thanks for pointing this out - I forgot to commit the final changes, sorry.

Reply:
The replacement `2023.3.0.dev` Sire packages are now available.